### PR TITLE
[wasm] Update repository URL in package.json

### PR DESCRIPTION
**Pull request by maraf**
Git protocol is obsolete.

### CODE REVIEW

File: src/mono/wasm/runtime/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": ".NET is a developer platform with tools and libraries for building any type of app, including web, mobile, desktop, games, IoT, cloud, and microservices.",
   "repository": {
     "type": "git",
-    "url": "git@github.com:dotnet/runtime.git"
+    "url": "https://github.com/dotnet/runtime"
```
Comment (teo-tsirpanis) [inline src/mono/wasm/runtime/package.json:6]:
```suggestion
    "url": "https://github.com/dotnet/runtime.git"
```

The `.git` suffix in the URL to clone is still being used.

image

Comment (maraf) [reply src/mono/wasm/runtime/package.json]: @teo-tsirpanis Do you know if there is any difference to using URL without `.git` suffix? Both of them works...

Comment (teo-tsirpanis) [reply src/mono/wasm/runtime/package.json]: Don't know, guess it's just a convention.

**Review by pavelsavara — APPROVED** [review]

File: src/mono/wasm/runtime/package.json (hunk 1 above)
Comment (maraf) [reply src/mono/wasm/runtime/package.json]: Ok, thanks

**Pull request merged**